api/src/test/java/uk/gov/dhsc/htbhf/claimant/ScheduledServiceIntegrationTest.java

```diff
@@ -181,7 +185,7 @@ private void assertChildTurnsOneInFirstWeekEmailPersonalisationMap(PaymentCycle
                 .contains(formatVoucherAmount(4)); // child is turning one in the next payment cycle, so one voucher per week going forward
         assertThat(personalisationMap.get(PAYMENT_AMOUNT.getTemplateKeyName()))
                 .isEqualTo(formatVoucherAmount(5)); // two vouchers in first week plus one for weeks two, three and four
-        assertThat(personalisationMap.get(REGULAR_PAYMENT.getTemplateKeyName())).asString()
+        assertThat(personalisationMap.get(EmailTemplateKey.REGULAR_PAYMENT.getTemplateKeyName())).asString()
```

Comment:
Need to explicitly declare EmailTemplateKey here as REGULAR_PAYMENT is now  in two enums.